scanner/e2etests/helmchart/templates/db-deployment.yaml

```diff
@@ -0,0 +1,96 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.app.db.name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Values.app.db.name }}
+spec:
+  replicas: 1
+  minReadySeconds: 15
+  selector:
+    matchLabels:
+      app: {{ .Values.app.db.name }}
+  template:
+    metadata:
+      namespace: {{ .Release.Namespace }}
+      labels:
+        app: {{ .Values.app.db.name }}
+    spec:
+      securityContext:
+        fsGroup: 70  # Set the group ownership to the `postgres` group, see dockerfile.
+      initContainers:
+      - name: init-db
+        image: "{{ (printf "%s/%s:%s" .Values.image.repository .Values.app.db.name .Values.image.tag ) }}"
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: POSTGRES_PASSWORD_FILE
+          value: "/run/secrets/stackrox.io/secrets/password"
+        - name: ROX_SCANNER_DB_INIT
+          value: "true"
+        command:
+        - init-entrypoint.sh
+        resources:
+          limits:
+            cpu: 2
+            memory: 4Gi
+          requests:
+            cpu: 200m
+            memory: 200Mi
+        volumeMounts:
+        - name: data
+          mountPath: /var/lib/postgresql/data
+        - name: config
+          mountPath: /etc/stackrox.d/config/
+          readOnly: true
+        - name: tls-volume
+          mountPath: /run/secrets/stackrox.io/certs
+          readOnly: true
+        - name: password
+          mountPath: /run/secrets/stackrox.io/secrets
+          readOnly: true
+      containers:
+      - name: db
+        image: "{{ (printf "%s/%s:%s" .Values.image.repository .Values.app.db.name .Values.image.tag ) }}"
+        imagePullPolicy: IfNotPresent
+        ports:
+        - name: postgresql
+          protocol: TCP
+          containerPort: 5432
+        resources:
+          limits:
+            cpu: 2
+            memory: 4Gi
+          requests:
+            cpu: 200m
+            memory: 200Mi
+        volumeMounts:
```

Comment:
See https://github.com/stackrox/stackrox/pull/7706/files#diff-0edd3f7dd805de769d6bd354718ccf38f6ba35fb2b79e29367815d3c07cf353f for some of the differences to consider adding here